src/app/core/market/api/template/template.model.ts

```diff
@@ -142,4 +149,24 @@ export class Template {
     }
   }
 
+  shippingAmount(country: string): Amount {
```

Comment:
all this stuff belongs in the listing model.

It involves the country parameter, which only applies to listings..

Reply:
ohhk